custom_components/tuya_local/devices/clofte_duo_400_humidifier.yaml

```diff
@@ -0,0 +1,75 @@
+name: Humidifier
+products:
+  - id: aa8iuikeenvbsznx
+    manufacturer: CLOFTE
+    model: DUO 400
+    model_id: T3-CLOFTE
+entities:
+  - entity: fan
+    translation_only_key: fan_with_presets
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 111
+        type: integer
+        name: speed
+        range:
+          min: 0
+          max: 4
+  - entity: switch
+    name: Sleep
+    translation_key: sleep
+    dps:
+      - id: 101
+        name: switch
+        type: boolean
+  - entity: switch
+    name: Quiet
+    icon: mdi:volume-high
+    dps:
+      - id: 116
+        name: switch
+        type: boolean
+  - entity: switch
+    name: Drying
+    dps:
+      - id: 105
+        name: switch
+        type: boolean
+  - entity: sensor
+    name: PM2.5
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        unit: µg/m³
+  - entity: sensor
+    name: Water level
```

Comment:
Use `translation_key: water_level`, and preferable use `class: enum` with all the possible values listed in a mapping (and mapped to the translated "empty", "low", "medium", "high", "full" values)